packages/storybook/src/react-cardgroup.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import readme from '@frameless/components-css/card/README.md?raw';
+import { CardGroup } from '@frameless/components-react/src/CardGroup';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import '@frameless/components-css/card/index.scss';
+
+// adds Card to Storybook
+const meta = {
+  title: 'React Component/CardGroup',
+
+  id: 'react-cardgroup',
+
+  component: CardGroup,
+
+  argTypes: {
+    children: {
+      name: 'Content',
+
+      description: 'CardGroup',
+
+      type: {
+        name: 'string',
+        required: true,
+      },
+
+      defaultValue: '',
+    },
+  },
+
+  args: {},
+
+  tags: ['autodocs'],
+
+  parameters: {
+    docs: {
+      description: {
+        component: readme,
+      },
+    },
+  },
+} satisfies Meta<typeof CardGroup>;
+
+export default meta;
+
+// defines that this (part of the) story is documentation
+type Story = StoryObj<typeof meta>;
+
+// adds Frameless Card section to Card in Storybook
+export const Default: Story = {
+  name: 'Frameless CardGroup', // in sidenav shows Frameless Card below Docs
+  args: {},
+  parameters: {},
```

Comment:
En deze heeft ook nog `children` nodig denk ik

Reply:
Hier zitten er ook children in? of begrijp ik het fout?


```
export const Default: Story = {
  args: {
    children: (
      <>
        <Card heading="Design system" headingLevel={2}>
          <Paragraph>
            Herbruikbare componenten onafhankelijk van huisstijl, daar mag je ons voor wakker maken! Frameless heeft al
            aan meerdere white-label design systems mogen werken en is trots op onze bijdrage aan NL Design System.
          </Paragraph>
```